src/base/rss/rssdownloadrule.h

```diff
@@ -88,6 +88,9 @@ namespace Rss
         bool operator==(const DownloadRule &other) const;
 
     private:
+        bool matches(const QString &articleTitle, const QString &expression) const;
+
+    private:
```

Comment:
Works for me. I'll update the pull request taking into account all the comments (probably later today - working my paid job right now).